client/test/README.md

```diff
@@ -0,0 +1,46 @@
+## Single test runner
+
+Single test runner loads only test files that are listed in config.json file. Configuration reflects folders structure.
+
+### Extending config.json
+
+Example folder: `client/state/plugins/wporg/test/`
+
+Test files we want to add to the runner:
+* `reducer.js`
+* `test-actions`
+* `test-selectors`
```

Comment:
You chose the folder with the file names that should be rewritten :laughing: (consistency, matching tested files)